### ISSUE TITLE
change MA0006 to be more lax

### ISSUE THREAD

**Issue by srasch** [meziantou/Meziantou.Analyzer#180]
Comparisons with "" or string.Empty should not trigger.

```c#
string val = "foo";

if (val == "") { ... }
if (val == string.Empty) { ... }
```

**Comment by meziantou** [comment]
Thanks for your feedback!

This should be fixed by https://github.com/meziantou/Meziantou.Analyzer/commit/c333c9a116bbe7350e4b3a37e0bf215a158ea3ff. You can update to [1.0.536](https://www.nuget.org/packages/Meziantou.Analyzer/1.0.536) to get the change.
